apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.IntegrationTests/BindingTests.cs

```diff
@@ -272,5 +274,9 @@ private async Task TestBindNonNull<T>(SpannerDbType parameterType, T value, Func
         public async Task BindJsonEmptyArray() => await TestBindNonNull(
             SpannerDbType.ArrayOf(SpannerDbType.Json),
             new string[] { });
+
+        internal static void MaybeSkipIfOnEmulator(bool runningOnEmulator, SpannerDbType spannerDbType) =>
+            Skip.If(runningOnEmulator && BindUnsupportedNullData.Contains([spannerDbType]),
```

Comment:
Is `[spannerDbType]` a collection expression here, and if so, are we able to use those at the moment? (I thought we'd have to wait until we're using the .NET 8 SDK.) I may just be missing something though.

Reply:
Yes, I just happily accepted Visual Studio's suggestion for this.